reminder/database.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import annotations
+
+from datetime import datetime
+
+import enum
+from sqlalchemy import BigInteger, Column, Integer, String, DateTime, Enum
+from typing import List, Dict
+
+from database import database, session
+
+
+class ReminderStatus(enum.Enum):
+    WAITING: str = "WAITING"
+    REMINDED: str = "REMINDED"
+    FAILED: str = "FAILED"
+
+    def str_list() -> str:
+        return ", ".join([e.name for e in ReminderStatus])
+
+
+class ReminderItem(database.base):
+    """Represents a database Reminder item for :class:`Reminder` module.
+
+    Attributes:
+        idx: The database ID.
+        guild_id: ID of the guild.
+        author_id: User ID of reminder author.
+        remind_id: User ID of reminded user.
+        permalink: Message URL.
+        message: Reminder text (None if empty).
+        origin_date: Date of creation.
+        remind_date: Date for reminding
+        status: Status of reminder
+    """
+
+    __tablename__ = "reminder_reminder_reminderitem"
+
+    idx = Column(Integer, primary_key=True, autoincrement=True)
+    guild_id = Column(BigInteger, default=None)
+    author_id = Column(BigInteger, default=None)
+    remind_id = Column(BigInteger, default=None)
+    permalink = Column(String, default=None)
+    message = Column(String, default=None)
+    origin_date = Column(DateTime(timezone=True), default=None)
+    remind_date = Column(DateTime(timezone=True), default=None)
+    status = Column(Enum(ReminderStatus), default=ReminderStatus.WAITING)
+
+    @staticmethod
+    def add(
+        guild_id: int,
+        author_id: int,
```

Comment:
I would like to see discord objects being passed to the methods. 
Covertin them to IDs in the methods on the database side instead of on the module side would be more readable I believe.
This is already used in the Unverify and Message board modules.

It will also reduce the number of parameters because of that.